en/docs/tutorials/create-your-first-webhook.md

```diff
@@ -0,0 +1,397 @@
+# Create Your First Webhook
+
+Choreo’s low-code editor allows developers to easily design (and then implement) high-quality webhooks. To explore this capability, let's consider a scenario where a team of software engineers in an organization wants to be notified via email whenever someone creates a GitHub issue with the `bug` label in a specific repository. 
+
+In this tutorial, you will address this requirement by doing the following:
+
+- Design a webhook that addresses the described requirement and test it in the Web Editor (i.e., the editing tool of Choreo), and commit the changes so that they are also available in the Choreo Console.
+- Deploy the webhook you created to the development environment.
+- Integrate GitHub with Choreo by connecting the webhook that you created to the required GitHub repository.
+- Test the webhook.
+- Promote the webhook to the production environment.
+
+## Step 1: Develop
+
+To create a project, add a Webhook component to it, design the webhook, test it, and then commit it to make it available in the Choreo Console, follow these sub-steps:
+
+### Step 1.1: Create a project and add a Webhook component
+
+First, let's add a Webhook component as follows:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev).
+
+2. Expand the **Project** list and click **+ Create New**.
+
+    ![Create project](../assets/img/tutorials/rest-api/create-project.png){.cInlineImage-full}
+
+3. Enter a unique project name and a description. For this tutorial, let's enter the following values:
+
+    | **Field**       | **Value**             |
+    |-----------------|-----------------------|
+    | **Name**        | `IssueTracking`       |
+    | **Description** | `Track GitHub issues` |
+
+4. Click **Create**.
+
+    The **Components** page opens as shown in the below image.
+
+    ![Create component](../assets/img/tutorials/rest-api/create-component.png){.cInlineImage-full}
+
+5. On the Components page, click **+ Create**.
+
+6. Click **Webhook**.
+
+    ![Add Webhook component](../assets/img/tutorials/webhook/add-webhook-component.png){.cInlineImage-full}
+
+7. Click **Start from scratch**.
+
+8. Select **GitHub** as the event source.
+
+    ![Select GitHub event source](../assets/img/tutorials/webhook/select-github-event-source.png){.cInlineImage-half}
+
+9. In the **Webhook** dialog, enter information as follows:
+
+    1. In the **Webhook Name** field, enter a unique webhook name (e.g., `IssueTracking`).
+   
+    2. From the **Trigger Channel** list, select **Issue Service** so that the system can trigger your webhook based on a change it observes in GitHub issues (i.e., assigning a label in this scenario).
+   
+    3. You need to save your Webhook implementation in a GitHub repository from which Choreo can access it. You can use a Choreo-managed GitHub repository or one of your own GitHub repositories. In this tutorial, let's use a Choreo-managed GitHub repository.
```

Comment:
should we mention fact that we are going with the default selection? Something along the lines of `Let's use the Choreo-managed repository, which is the default selection.`

Reply:
Changed as suggested.